callautomation_appointmentbooking/resources/config.conf

```diff
@@ -0,0 +1,25 @@
+[APPOINTMENT_BOOKING_CONFIG]
+ACS_CONNECTION_STRING=<YOUR_ACS_CONNECTION_STRING>
+
+# public Callback URI that will be used to answer incoming call event,
+# or handle mid-call events, such as CallConnected.
+# See README file for details on how to setup tunnel on your localhost to handle this.
+BASE_CALLBACK_URI=<YOUR_BASE_CALLBACK_URI>
```

Comment:
Ask for a trailing slash as the code does not check for this and results in invalid URLs